models/orderDao.js

```diff
@@ -0,0 +1,168 @@
+        JOIN brands b ON p.brand_id = b.id
+        JOIN pay_status ps ON bsk.status_id = ps.id
+        WHERE u.id = '${extractedId}' AND ps.id = 1;       
+        `)
+        return result
+        
+    } catch (err) {
+        console.log(err)
+        middleErr.error(500 , 'INVALID_DATA_INPUT' )
+    }
+}
+
+//장바구니 상품 수량추가
+const addProduct = async (verifiedEmail, productId) => {
+    try {
+        const userId = await appDataSource.query(`
+        select id from users where email = '${verifiedEmail}'
+        `)
+        const extId = userId[0].id
+        const product_id = await appDataSource.query(`
+        select id from products where id = '${productId}'
+        `)
+        const extProduct_id = product_id[0].id
+        
+         const result =  await appDataSource.query(`
+                UPDATE basket
+                SET basket.quantity = basket.quantity + 1
+                WHERE basket.user_id = '${extId}' and basket.product_id = '${extProduct_id}'
+        `)
+    } catch (err) {
+        console.log(err)
+        const error = new Error('INVALID_DATA_INPUT')
+        error.statusCode = 500;
+        throw error
+    }
+};
+// 장바구니 상품 삭감
+const cutProduct = async(verifiedEmail,productId) => {
+    try{
+        const userId = await appDataSource.query(`
+        select id from users where email = '${verifiedEmail}'
+        `)
+        const extractedId = userId[0].id
+        const product_id = await appDataSource.query(`
+        select id from products where id = '${productId}'
+        `)
+        const extProduct_id = product_id[0].id
+        await appDataSource.query(`
+        update basket
+        set basket.quantity = basket.quantity - 1
+        where basket.user_id = '${extractedId}' and basket.product_id = '${extProduct_id}'
+        `)
+    
+    }catch(err){
+        console.log(err)
+        middleErr.error(500 , 'INVALID_DATA_INPUT' )
+    }
+};
+
+//장바구니 상품 삭제
+const deleteProduct = async(verifiedId, productId)=>{
+    try{
+        const userId = await appDataSource.query(`
+        select id from users where id = '${verifiedId}'
+        `)
+        const extId = userId[0].id
+        const product_id = await appDataSource.query(`
+        select id from products where id = '${productId}'
+        `)
+        const extProductId = product_id[0].id
+        return await appDataSource.query(`
+        DELETE FROM basket
+        WHERE user_id = '${extId}' AND product_id = '${extProductId}';
+        `)
+    }catch(err){
+        console.log(err)
+        middleErr.error(500, 'INVALID_DATA_INPUT')
+    }
+};
+
+// 장바구니 담기
+const addBusket = async(verifiedUserId, product_id)=>{
+    try{
+        return await appDataSource.query(`
+        insert into basket(user_id, product_id, quantity, status_id)
+        values (?,?,?,?)
+        `, [verifiedUserId, product_id, 1, 1])
+        
+    }catch(err){
+        console.log(err)
+        middleErr.error(500, 'INVALID_DATA_INPUT')
+    }
+};
```

Comment:
```suggestion
// 장바구니 담기
const basketStatusType = {
  'PENDING': 1
}
const createBasket = async(userId, productId) => {
    try {
        return await appDataSource.query(`
        INSERT INTO basket(
          user_id, 
          product_id, 
          quantity, 
          status_id
        )
        values (?, ?, ?, ?)
        `, [verifiedUserId, product_id, 1, basketStatusType.PENDING])
        
    } catch(err) {
        console.log(err)
        middleErr.error(500, 'INVALID_DATA_INPUT')
    }
};
```

1. 테이블 이름이 `Basket`이므로 함수 이름도 맞춰져 있어야 합니다.
2. SQL문은 대문자로 작성해서 구문의 가독성을 높여주세요 :)
3. (질문) `status_id` 1번은 어떤 상황인가요? 개발자들이 상태를 외우고 있어야 하는 불편함이 있을 수 있습니다. status를 관리할 수 있는 type객체를 하나 만들어서 관리해주시면 의미 전달에 보다 편리합니다.